.env.test

```diff
@@ -10,7 +10,9 @@ DQT_CLIENT_PORT=13
 ADMIN_ALLOWED_IPS=::1,127.0.0.1
 
 GECKOBOARD_API_KEY=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
-ENVIRONMENT_NAME=test
+ENVIRONMENT_NAME=development
+
+RAILS_ENV=development
```

Comment:
@nomasprime @and-lucas-kelly Should these not be test?

Reply:
it should be test